go.mod

```diff
@@ -1,122 +1,131 @@
-go 1.17
+go 1.18
 
-module github.com/cosmos/ibc-go/v3
```

Comment:
It's just a specifier indicating the path and version for that module. Also, for line/ibc-go, I thought that v3 had no meaning, so I removed it.

Reply:
I think it has some reason, because each version of `cosmos/ibc-go` has each module path. 
https://github.com/cosmos/ibc-go/blob/8f0cfb0682bb6dc6cf311d4644b300041e277866/go.mod#L3